public/javascripts/data/user.js

```diff
@@ -2,24 +2,25 @@ const Database = require('better-sqlite3');
 const hashing = require('../hashing')
 
 
-
+// Function checks if user inputted username is available
 exports.signUp = (username) => {
```

Comment:
This method would be better named `usernameAvailable` or similar - It's not actually signing up the user. It seems like that bit is taking place in the `writeUserToDatabase` method